### ISSUE TITLE
Pre-training under infinite compute

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#413]
## Links
https://arxiv.org/abs/2509.14786

- 本論文は、計算資源が無限だが学習データが有限という将来シナリオでの言語モデル事前学習の方法を探る。
- 標準的なエポック増加やパラメータ増加では過学習が生じるが、正則化・アンサンブル・蒸留を組み合わせることで大幅なデータ効率改善を実証。
- 最適化によりデータ効率は最大17.5倍向上し、下流タスクにも性能改善が転移することを確認した。

### 問題設定
- データの増加速度（年1.03倍）が計算資源の増加速度（年4倍）に追いつかない未来を想定
- 200Mトークンという制約データセットで実験を行い、計算資源に制約を設けない環境での最適学習戦略を探った。

### 既存手法の限界

- エポック数やパラメータ数を増やす標準的な方法は、一定の規模を超えると検証損失が上昇＝過学習に陥る（図2, p.4）

### 正則化の効果

- 標準的な weight decay (0.1) は不十分で、30倍大きな weight decay が必要と判明。
- 適切な正則化により、パラメータ数に対して損失が単調に減少する「きれいなスケーリング則」が成立（図3, p.5）。

### アンサンブルの活用

- 複数の中規模モデルを独立に学習させ、その出力を平均することで、大規模単一モデルよりも低い漸近損失を達成（図4, p.7）。
- K→∞でのアンサンブルの漸近値は、N→∞での単一モデルより優れる。

### 正則化とアンサンブルの統合

- 二重極限（N, K→∞）を取ることで、最良の損失 3.17 を達成（図6, p.9）。これは標準手法の3.75より大幅に改善。

### データ効率性の実証

- 標準手法に比べ、正則化＋アンサンブルの統合手法は 5.17倍のデータ効率 を実現（図8, p.11）。
- 効率性の改善は200Mトークンに限らず、大規模トークンでも持続することをデータスケーリング則で確認。

### 蒸留による小型化

- アンサンブル（例：300Mモデル×8）を蒸留すると、8倍小さい単一モデルに83%の性能改善を保持（図9, p.13）。
- 自己蒸留（教師と生徒が同じモデル規模）でも性能改善が見られ、過学習を避けつつ効率を高められる。

### 下流タスクへの転移

- PIQA, SciQ, ARC Easy などのベンチマークで9%以上の精度向上（図10, p.14）。
- 数学データセット（MegaMath-Web-Pro）での継続学習においても、73Bトークン学習に匹敵する性能を4Bトークンで達成＝17.5倍の効率改善（表1, p.15）。

## 新規性・これまでの研究との差異

- 既存の研究は計算資源制約下でのスケーリング則（Chinchilla, Kaplanら）を前提としていたが、本論文はデータ制約下での無限計算資源シナリオを対象とする点が新しい。
- 標準的な「エポックやパラメータを増やす」アプローチでは成立しないスケーリング則を、正則化・アンサンブルにより再構築。
- 損失の比較を「有限計算量での性能」ではなく、スケーリング則の漸近値で評価する新しい基準を提案。

## 貢献の実証方法

- 検証損失の漸近挙動を複数のスケールでフィッティングし、理論的限界を推定。
- 下流タスクの性能評価で、正則化・アンサンブル・蒸留の改善が転移することを確認。
- **継続事前学習（CPT）**に適用し、実データ効率の大幅改善を示した。
- すべての実験コード・結果をオープンソース化し、再現性を確保。
